sdk/daml-lf/snapshot/scripts/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pandas==2.2.1
```

Comment:
Modifying `shell.nix` to load these packages via nix wasn't working for myself. Hence, capturing these dependencies here so that we can manually install them via `pip3 install -r requirements.txt` in CI-like environments.